skimage/transform/tests/test_geometric.py

```diff
@@ -1023,9 +1114,45 @@ def test_affine_transform_from_linearized_parameters():
         _ = AffineTransform(matrix=v[:-1])
 
 
-def test_affine_params_nD_error():
+EG_OPS = dict(scale=(4, 5), shear=(1.4, 1.8), rotation=0.4, translation=(10, 12))
+
+
+@pytest.mark.parametrize(
+    'tform_class,op_order',
```

Comment:
```suggestion
    'tform_class, op_order',
```
Same question, not sure...